include/crm/services_internal.h

```diff
@@ -38,8 +38,18 @@ svc_action_t *services__create_resource_action(const char *name, const char *sta
                                       int timeout /* ms */, GHashTable *params,
                                       enum svc_action_flags flags);
 
+/*!
+ * \brief Set operation rc and status per errno from stat(), fork() or execvp()
+ *
+ * \param[in,out] op     Operation to set rc and status for
+ * \param[in]     error  Value of errno after system call
+ *
+ * \return void
+ */
+void services__handle_exec_error(svc_action_t * op, int error);
+
```

Comment:
Using the double underbar is a good idea, but this was fine to leave in services_private.h. The \*_private.h headers are library-scoped (i.e. only code in the library itself uses them), so they can be marked as G_GNUC_INTERNAL, which lets the linker be a little more efficient. The \*_internal.h headers are internal to all of pacemaker (i.e. pacemaker code outside the library can use them).